### DISCUSSION TITLE
Move the test helper code into lib/

### DISCUSSION THREAD

**Discussion by drnic** [josefarias/hotwire_combobox#217] [Ideas]
To use combobox in my app I needed to copy+paste two of the library's own test helper files into my test folder (and the `#clickable` box into my layout)

Would this be useful to others if these files were part of the repo's lib/ folder and more easily reusable in apps?

**Comment by josefarias** [comment]
Hey @drnic!

Just wanna make sure I understand. This wasn't an impediment to _using_ the combobox in the app, correct? I take it you were looking to test some flows that included the combobox, and thats when you needed some helpers, which you copied from the gem's test suite?

If I understood correctly — I don't know if I want to officially support test helpers. I haven't given it much thought!

Gut reaction is probably no. Then we'd have to make sure they're properly kept up to date and working with any common test set ups. I think I'd rather let the interested users look in the code and copy these over to their apps as needed, and hopefully make suggestions for the gem's suite as well if they have any. That way we could officially support test helpers if we had certainty they're robust enough.

**Reply by drnic** [reply]
Correct, just discussing how apps manipulate the combo boxes  and assert things about them.
